# Message that contains a trajectory for mobile robot navigation

std_msgs/Header header
hector_nav2/TrajectoryPointMsg[] trajectory


